### PR TITLE
chore(go): generate module embedding

### PR DESCRIPTION
**Pull request by SoManyHs**
Closes https://github.com/aws/jsii/issues/2008

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

File: packages/jsii-pacmak/lib/targets/go.ts [hunk 1]
```diff
@@ -2,11 +2,12 @@ import { CodeMaker } from 'codemaker';
 import { Documentation } from './go/documentation';
 import { Assembly } from 'jsii-reflect';
 import { Rosetta } from 'jsii-rosetta';
-import { join } from 'path';
 import { RootPackage } from './go/package';
 import { IGenerator } from '../generator';
 import { Target, TargetOptions } from '../target';
 import { goPackageName } from './go/util';
+import * as fs from 'fs-extra';
+import * as path from 'path';
```
Comment (RomainMuller) [inline packages/jsii-pacmak/lib/targets/go.ts:10]: May I be so bold as to ask we sort those imports (external / not-starting with `.` first, then internal / starting with `.`)?

File: packages/jsii-pacmak/lib/targets/go.ts [hunk 2]
```diff
@@ -53,23 +54,36 @@ class GoGenerator implements IGenerator {
     });
   }
 
-  public async save(outDir: string, _tarball: string): Promise<any> {
+  public async save(outDir: string, tarball: string): Promise<any> {
     // TODO: Replace with [go-embed](https://github.com/pyros2097/go-embed)
-    // const bundledRuntimeDotGo = path.join('_jsii', 'bundled-runtime.go');
-    // this.code.openFile(bundledRuntimeDotGo);
+    const bundledRuntimeDotGo = path.join('_jsii', 'bundled-runtime.go');
+    this.code.openFile(bundledRuntimeDotGo);
```
Comment (RomainMuller) [inline packages/jsii-pacmak/lib/targets/go.ts:60]: This file was opened, but never closed... Perhaps this explains why it does not show in the output?

Comment (MrArnoldPalmer) [inline packages/jsii-pacmak/lib/targets/go.ts:60]: These two lines aren't doing anything, they can be removed.

**Review by MrArnoldPalmer — CHANGES_REQUESTED** [review]

**Review by MrArnoldPalmer — APPROVED** [review]
👏

**Pull request merged**